src/main/resources/application.yml

```diff
@@ -13,13 +13,13 @@ spring:
     serialization:
       WRITE_DATES_AS_TIMESTAMPS: false
 
-  # TODO: This security section can be removed if your service doesn't call out to other services
```

Comment:
I don't think this is strictly true (we still need the `resourceserver` configuration if the kotlin project is functioning as an API don't we?)